webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -76,6 +76,8 @@ var userOverride = {
         voiceOutput: false,
         capturePhoto: false,
         captureVideo: false,
+        maxAttachmentSize: 25, // default size is 25MB
+        maxAttachmentSizeErrorMsg: "File size cannot be more than $maxAttachmentSize MB"
```

Comment:
this must come from MCK_LABELS where customer can put it in different language or different text

Reply:
Yes sir, I missed it.
Let me correct